bndtools.core/bndtools.gtk.linux.x86_64.bndrun

```diff
@@ -1,15 +1,24 @@
 -include: bndtools.shared.bndrun
 
 -runbundles.linux: \
+	org.eclipse.swt.gtk.linux.x86_64,\
+	org.eclipse.e4.ui.workbench.renderers.swt,\
```

Comment:
this is already in the shared bndrun, so don't duplicate it here.

Reply:
right.